x/profiles/client/rest/query.go

```diff
@@ -15,6 +15,23 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/profiles/parameters", queryProfilesParamsHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/profiles/{address_or_dtag}", queryProfileHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/profiles", queryProfilesHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/dtag/requests/{address}", queryDTagRequests(cliCtx)).Methods("GET")
```

Comment:
I personally think it's better to change this endpoint making it more similar to the `profile` one: 

```suggestion
	r.HandleFunc("/profiles/{address}/dtag-transfer-requests", queryDTagRequests(cliCtx)).Methods("GET")
```